examples/datasource-basic/README.md

```diff
@@ -57,4 +58,4 @@ A data source backend plugin consists of both frontend and backend components.
 
 - [Grafana plugins documentation](https://grafana.com/developers/plugin-tools/)
 - [Build a data source plugin tutorial](https://grafana.com/developers/plugin-tools/tutorials/build-a-data-source-plugin)
-- [Grafana UI Library](https://developers.grafana.com/ui) - UI components to help you build interfaces using Grafana Design System
+- [Grafana documentation](https://grafana.com/docs/)
```

Comment:
Linking to the UI components is generally useful

Reply:
done